tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1197,7 +1239,8 @@ def runTest(self):
                             pkts_num_leak_out + pkts_num_trig_pfc) // cell_occupancy - 1 - margin)
             elif 'cisco-8000' in asic_type:
                 fill_leakout_plus_one(self, src_port_id, dst_port_id,
-                       pkt, int(self.test_params['pg']), asic_type)
```

Comment:
======================================================================
ERROR: sai_qos_tests.PFCtest
----------------------------------------------------------------------
Traceback (most recent call last):
  File \"saitests/py3/sai_qos_tests.py\", line 1249, in runTest
    pkt, int(self.test_params['pg']), asic_type, pkts_num_egr_mem)
UnboundLocalError: local variable 'pkts_num_egr_mem' referenced before assignment